src/client.rs

```diff
@@ -25,40 +25,58 @@ const BACKGROUND_SENDER: &str = "Background Sender";
 #[derive(Clone)]
 pub struct Client {
     cmd_tx: Sender<Command>,
+    /// The runtime parameters for this test.
+    pub run_parameters: RunParameters,
```

Comment:
```suggestion
    run_parameters: RunParameters,
```

How about, instead of making these public, offering a getter method for them? Would (a) prevent misuse and thus subtle bugs in client code and (b) convey the fact that these are read-only.

Reply:
You are right, thanks!